src/main/mate.academy/dao/Validator.java

```diff
@@ -0,0 +1,5 @@
+package dao;
+
+public interface Validator {
+    void validator(String[] fileDataLine);
```

Comment:
```suggestion
    void validate(String[] fileDataLine);
```
method names should be verbs